juju/python-libjuju
# Target ceiling version

#### Description

Currently 

1. the “target” version is hardcoded, 
2. we spam debug output for any use of pylibjuju where the target version != juju controller version. 

This reads the version from the `VERSION` file we keep at the toplevel. And removes the minor version sniffing.

We still keep the major version sniffing because of the two tracks we have in pylibjuju (2.9 and 3.x).

#### QA Steps

No added functionality, so no QA is needed other than reading the code.

If you wanna be really pedantic, set the logging to DEBUG and run the line above, and you shouldn't see any messages like 

```
"This version was tested using ...... juju version ...... may have compatibility issues"
```

#### Notes & Discussion

JUJU-5315
